package.json

```diff
@@ -1,12 +1,13 @@
 {
   "name": "@phyloref/phyx",
-  "version": "0.1.2",
+  "version": "0.2.0",
   "description": "Phyloreference Exchange (PHYX) library in JavaScript",
```

Comment:
BTW that this is JavaScript shouldn't need mentioning in the description. Perhaps circumscribe what this library allows doing with PHYX?

I realize this line isn't part of the diff, noting it here anyway. Feel free to move to a new issue.